install/installer/pkg/config/v1/config.go

```diff
@@ -153,6 +153,8 @@ type ObjectStorageS3 struct {
 type ObjectStorageCloudStorage struct {
 	ServiceAccount ObjectRef `json:"serviceAccount" validate:"required"`
 	Project        string    `json:"project" validate:"required"`
+	// BucketPrefix is an optional prefix that's prepended to all bucket names
+	BucketPrefix   string    `json:"bucketPrefix,omitempty"`
```

Comment:
!! This is a backwards incompatible change. If a (self-hosted) user were to upgrade with this new version they would not find their backups anymore (`stage` was fixed to `production` before).

Please make sure that we maintain backwards compatibility.

Reply:
Exactly, I'm aware. That's why I fixed that to "dev" (should indeed be "production").
The exact way how to ensure backwards compatibility will be discussed with Self-Hosted next week. Until then this will remain a draft.